c7n/policy.py

```diff
@@ -838,6 +821,56 @@ def get_session_factory(provider_name, options):
             "%s provider not installed" % provider_name)
 
 
+class PolicyConditions(object):
+
+    filter_registry = FilterRegistry('c7n.policy.filters')
+
+    def __init__(self, policy, data):
+        self.policy = policy
+        self.data = data
+        self.filters = []
+
+    def validate(self):
+        self.filters.extend(self.convert_deprecated())
+        self.filters = self.filter_registry.parse(
+            self.filters, self.policy.resource_manager)
+
+    def evaluate(self):
+        policy_vars = {
+            'region': self.policy.options.region,
+            'resource': self.policy.resource_type,
+            'provider': self.policy.provider_name,
+            'account_id': self.policy.options.account_id,
```

Comment:
Is it possible to add in 'account' here as well so the account name can be filtered on as well?
```
'account': self.policy.options.account,
```

Reply:
i'd like to avoid it for now, we don't generally do api calls pre-policy execution (one exception atm re region expansion for `all`), there's also been a slow proliferation of account name acquisition in various filters/actions (invoke, notify, etc), which would be better to tackle more holistically rather than adding on to the pile, likely via a thread local credential attribute similiar to local session. for now though, avoiding account name is the simplest option. note a separate ability to do constant checks in value type filters would enable account name usage with c7n-org sans api calls.

Reply:
ok sounds good and makes sense.  Thanks for the explanation

Reply:
so if your running with c7n-org this will now pull in the account info (including name) into an account dictionary that can be filtered against.